src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -2808,4 +2808,25 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Convert_to_record" xml:space="preserve">
     <value>Convert to record</value>
   </data>
+  <data name="Introduce_parameter" xml:space="preserve">
+    <value>Introduce parameter</value>
+  </data>
+  <data name="Introduce_parameter_for_0" xml:space="preserve">
+    <value>Introduce parameter for '{0}'</value>
+  </data>
+  <data name="Introduce_parameter_for_all_occurrences_of_0" xml:space="preserve">
+    <value>Introduce parameter for all occurrences of '{0}'</value>
+  </data>
+  <data name="Introduce_new_parameter_overload_for_0" xml:space="preserve">
+    <value>Introduce new parameter overload for '{0}'</value>
+  </data>
+  <data name="Introduce_new_parameter_overload_for_all_occurrences_of_0" xml:space="preserve">
+    <value>Introduce new parameter overload for all occurrences of '{0}'</value>
+  </data>
+  <data name="Introduce_parameter_and_extract_method_for_0" xml:space="preserve">
+    <value>Introduce parameter and extract method for '{0}'</value>
+  </data>
+  <data name="Introduce_parameter_and_extract_method_for_all_occurrences_of_0" xml:space="preserve">
+    <value>Introduce parameter and extract method for all occurrences of '{0}'</value>
```

Comment:
can you show me what this looks like in the lightbulb?

Reply:
![image](https://user-images.githubusercontent.com/40616383/112224074-09eafa00-8be8-11eb-854d-4cbd7a9de4c1.png)

Reply:
Since you always have the top-level 'Introduce parameter'. we can greatly decrease the repetition of the individual items, and we can clarify what they do.  So we can have:

```
Introduce parameter for '{0}'
    and inline it into all callsites
    into new overload
    into new method invoked from callsites
```

Reply:
Sounds good, changed it